lib/traces/backend/newrelic/version.rb

```diff
@@ -0,0 +1,11 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/traces-backend-newrelic/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module Traces
+  module Backend
+    module NewRelic
+      VERSION = '0.0.1'
```

Comment:
```suggestion
      VERSION = '0.1.0'
```

Since the third column is for patch releases, I think moving the "1" to the minor release column makes more sense. It also matches the version for the OTel gem.